source/NVDAObjects/__init__.py

```diff
@@ -1267,3 +1268,35 @@ def getSelectedItemsCount(self,maxCount=2):
 		For performance, this method will only count up to the given maxCount number, and if there is one more above that, then sys.maxint is returned stating that many items are selected.
 		"""
 		return 0
+
+	reviewBounded = False
+	
+	@script(gesture="kb:NVDA+o",
+		# Hardcoded to avoid circular import
+		category=_(u"Text review"),
+		# Translators: A gesture description.
+		description=_(u"Toggles whether review is constrained to the currently visible text"))
+	def script_toggleReviewBounds(self, gesture):

```

Comment:
It worries me that this is is on the object instead of a global command. Is there a particular reason for that?

Reply:
I too think this could be cleaner as a global command, but the `reviewBounded` variable is also on the object so keeping this there might be better from an organizational standpoint...

Reply:
Question is whether we really believe that review Bounded should be on the object or on the textInfo, or even global. If an object gets destroyed and created again, the state of review Bounded is lost. Having review bounds on tree interceptors also makes sense, i.e. when you want to review with document review what is on screen on a particular page.

I also think that it doesn't make much sense to have review bounds doing anything when in screen review. So in that case, the command should just do nothing.

Reply:
Yes, having this as a global default makes sense to me.

Reply:
Yes, I also think a global default would be a plus.
It does not make much sense for a "screen" review to easily review what's not on screen.
It can be helpful at times, but is most often confusing when collaborating with a sighted person.

Reply:
This has now been made a global default.